docs/API.REST.md

```diff
@@ -697,3 +699,122 @@ Bulk import only works on documents in our persistent data storage layer.
   }
 }
 ```
+
+---
+
+### Getting the last statistics frame
+
+Kuzzle monitors its internal activities and make snapshots of them. This command allows getting the last stored statistics frame.
+
+These statistics include:
+
+* the number of connected users for protocols allowing this notion (websocket, udp, ...)
+* the number of ongoing requests
+* the number of completed requests since the last frame
+* the number of failed requests since the last frame
+
+**URL:** ``http://kuzzle:7512/api/_getStats``
+
+**Method:** ``GET``
+
+**Response:**
+
+```javascript
+{
+  error: null,                      // Assuming everything went well
+  result: {
+    count: <number of found documents>
+    _source: {                      // Your original count query
+      ...
+    },
+    collection: '<data collection>',
+    action: 'getStats',
+    controller: 'admin',
+    statistics: {
+      "YYYY-MM-DDTHH:mm:ss.mmmZ": {
```

Comment:
I'm not sure about the date is the key for the nested object. That force in js to make something like `Object.keys()` for retrieve the date and for access to the embed object. Or maybe I miss something ?